### PR TITLE
Fixed default LogLevel for the Skip mode

### PR DESCRIPTION
**Pull request by dtrabo**
The default Log Level for Skip Startup mode was Error instead of Verbose

### CODE REVIEW

**Review by guperrot — DISMISSED** [review]

File: Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original [hunk 1]
```diff
@@ -61,9 +61,9 @@ static const int kMSStartupType = 0 /*STARTUP_TYPE*/;
       [[NSUserDefaults standardUserDefaults] objectForKey:kMSStartTargetKey];
     int startTargetValue =
       startTarget == nil ? kMSStartupType : startTarget.intValue;
+    [MSAppCenter setLogLevel:(MSLogLevel)kMSLogLevel];
     if (startTargetValue == SKIP) {
         return;
-      
   }
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original:64]: Please format

**Review by Jamminroot — DISMISSED** [review]
Aside from what @MatkovIvan pointed out, LGTM

**Review by thyeggman — DISMISSED** [review]

File: Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original [hunk 2]
```diff
@@ -57,32 +57,32 @@ static const int kMSStartupType = 0 /*STARTUP_TYPE*/;
 }
 
 + (void)startAppCenter {
-    NSNumber *startTarget =
-      [[NSUserDefaults standardUserDefaults] objectForKey:kMSStartTargetKey];
+  NSNumber *startTarget =
+    [[NSUserDefaults standardUserDefaults] objectForKey:kMSStartTargetKey];
     int startTargetValue =
       startTarget == nil ? kMSStartupType : startTarget.intValue;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Core/AppCenterStarter.original]: I'm sorry, but the formatting still looks wrong.

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by MatkovIvan — COMMENTED** [review]
Please update changelog

**Review by guperrot — DISMISSED** [review]

**Review by Jamminroot — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ Updated native SDK versions:
 
 * **[Feature]**  Add `Distribute.setEnabledForDebuggableBuild(boolean)` method to allow in-app updates in debuggable builds.
 * **[Bug fix]**  Fixed UWP build in Unity 2019
+* **[Bug fix]**  Fixed wrong default Log Level value for the test app.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: Please move it to the correct version

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ Updated native SDK versions:
 
 * **[Feature]**  Add `Distribute.setEnabledForDebuggableBuild(boolean)` method to allow in-app updates in debuggable builds.
 * **[Bug fix]**  Fixed UWP build in Unity 2019
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
 * **[Bug fix]** Fixed UWP build in Unity 2019.
```

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Comment by annakocheshkova** [comment]
We shouldn't update changelog. This is a test app issue.
Customer can't choose any of the startup modes, this is done implicitly and it worked already.
Please revert the changes in the changelog.

**Comment by MatkovIvan** [comment]
:thinking: okay, @annakocheshkova you are right. It's a bit weird since it modifies the file in the SDK, not in test app.

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**